src/sdk/modules/smartSessionsValidator/toSmartSessionsValidator.ts

```diff
@@ -123,14 +123,14 @@ export const toSmartSessionsValidator = (
     getStubSignature: async () =>
       encodeSmartSessionSignature({
         mode,
-        permissionId,
+        permissionId: permissionIds[permissionIdIndex],
```

Comment:
when getting a sig we only need one permissionId at a time. which represents a session

Reply:
need to check if there is case where it's making use of multiple already-enabled sessions in a single userop.

Reply:
https://github.com/bcnmy/sdk/issues/116

Reply:
In this pr the functionality hasn't changed. I've only paved the way for accommodating multiple permissionIds later.

Reply:
If you'd prefer I can revert, but I don't see any harm in doing this now.

Reply:
hmm the module keeps active permission id single because for signature only one can be used at a time. 
reverting is better only because we first check feasibility, ask @fil then make changes.